tests/CommunityToolkit.Aspire.Hosting.ActiveMQ.Tests/AppHostTests.cs

```diff
@@ -0,0 +1,43 @@
+using Aspire.Components.Common.Tests;
+using CommunityToolkit.Aspire.Testing;
+using FluentAssertions;
+
+namespace CommunityToolkit.Aspire.Hosting.ActiveMQ.Tests;
+
+[RequiresDocker]
+public class AppHostTests(AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_ActiveMQ_MassTransit> fixture) : IClassFixture<AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_ActiveMQ_MassTransit>>
+{
+    [Fact]
+    public async Task ResourceStartsAndRespondsOk()
+    {
+        var resourceName = "masstransitExample";
+        await fixture.ResourceNotificationService.WaitForResourceHealthyAsync(resourceName).WaitAsync(TimeSpan.FromMinutes(2));
+        await Task.Delay(15000);
```

Comment:
We shouldn't put a `Task.Delay`, this suggests that there's something else we should be doing to report the status of the resource, possibly a health check that will then hold the `WaitForResourceHealthy` to complete.

Reply:
I added a healthcheck to the MassTransit example application. I used this to wait for health. I also tried to implement a unittest to check if a message-received counter is increased or not. I needed to add a retry in this unittest. Is this acceptable?

After this I will try to implement a HealthCheck on the ActiveMQ host. Maybe I will ask for advice.